src/workspace/src/store.ts

```diff
@@ -5,36 +5,37 @@ import {
   StorefrontNavigationSection,
```

Comment:
this is mostly formatting updates .. amongst this are small renaming, and adding RT Credit to the Apps list